### DISCUSSION TITLE
With Skip 1.0.0 - should I still see

### DISCUSSION THREAD

**Discussion by davidakoontz** [skiptools/skip#205] [Q&A] [answered]
I installed Skip preRelease - then updated.  Should I see Skip 0.10.8 init  OR should it be Skip 1.0.0 init?

<img width="813" alt="Screenshot 2024-08-23 at 11 02 29 AM" src="https://github.com/user-attachments/assets/7525c285-3ffc-427f-a2bb-8f6e9207a3be">

**Comment by davidakoontz** [comment]
It would appear that Skip checkup does not think I updated to 1.0.0.

<img width="1072" alt="Screenshot 2024-08-23 at 11 21 54 AM" src="https://github.com/user-attachments/assets/06e24bf3-9e1b-4510-895f-fc5276aba84e">

**Comment by davidakoontz** [comment]
After a proper update:  **skip upgrade**

I'm seeing 1.0.6 in the skip version - that feels better.

<img width="830" alt="Screenshot 2024-08-23 at 11 25 43 AM" src="https://github.com/user-attachments/assets/14e0b7a8-5e04-465d-8af5-9ae67c2bc3b5">

**Comment by marcprux** [comment, marked as answer]
Yes, for the Skip 1.0 upgrade, you do need to manually do `skip upgrade`. Glad that fixed it!
